### PR TITLE
Create 2016-12-08-the-need-for-speed.html.md

### PR DESCRIPTION
**Pull request by ifireball**
Adding a blog post about recent change to the CI system that the developers should be aware of.

### CODE REVIEW

**Comment by duck-rh** [comment]
Interesting title; is this related to personal interests? ;-P

File: source/blog/2016-12-08-the-need-for-speed.html.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+READMORE
+
+From the oVirt infra team's point of view, the CI standards have removed the need to constantly maintain build 
+dependencies on the Jenkins slaves and also eliminated most of the situations where jobs running on the same
+slave influenced one another.
+
+The CI standards implementation we have has one shortcoming, it was not particularity fast.
+
+We started seriously looking at this after one of the _VDSM_ maintainers [complained](http://lists.ovirt.org/pipermail/devel/2016-December/014427.html) that the "_check_patch_" jobs
+for his project are running for far to long a time. It the end it turned out that a major reason for the delay
```
Comment (mykaul) [inline source/blog/2016-12-08-the-need-for-speed.html.md]: It the end -> In the end

**Comment by ifireball** [comment]
@duck-rh I'm not really a gamer, but gaming titles usually attract readers ;)
(Case to point - the staggering amount of Docker tutorials for running Minecraft servers...)

**Comment by eedri** [comment]
:+1:

**Comment by bproffitt** [comment]
Is this ready now? Or did you want to post it tomorrow as the date in the filename implies?

**Comment by ifireball** [comment]
It already the 8th in my corner of the globe, so fire at will...

**Pull request merged**